### ISSUE TITLE
Xonsh eats The program x is currently not installed. You can install it by typing: sudo apt install output

### ISSUE THREAD

**Issue by minecraft2048** [xonsh/xonsh#1331]
Lets say I try to run a program that doesn't exist in my computer, for example sql. Here is what fish does:

```
feanor@silmaril ~> sql
The program 'sql' is currently not installed. You can install it by typing:
sudo apt install parallel
feanor@silmaril ~> 
```

This is what Xonsh does:

```
feanor@silmaril ~ $ sql
xonsh: subprocess mode: command not found: sql
Did you mean one of the following?
    l:   Alias
    ss:  Command (/bin/ss)
    ll:  Alias
    su:  Command (/bin/su)
    sn:  Command (/usr/bin/sn)

The program 'sql' is currently not installed. You can install it by typing:
feanor@silmaril ~ $ 
```
The difference is that in fish and bash, there the `sudo apt install parallel` , while Xonsh doesn't have that

Here is my xonfig:
```
feanor@silmaril ~ $ xonfig
+------------------+----------------+
| xonsh            | 0.3.4          |
| Python           | 3.5.1          |
| PLY              | 3.7            |
| have readline    | True           |
| prompt toolkit   | 1.0.0          |
| shell type       | prompt_toolkit |
| pygments         | 2.1.3          |
| on posix         | True           |
| on linux         | True           |
| distro           | Ubuntu         |
| on darwin        | False          |
| on windows       | False          |
| on cygwin        | False          |
| is superuser     | False          |
| default encoding | utf-8          |
+------------------+----------------+
feanor@silmaril ~ $ 
```

**Comment by laerus** [comment]
Hello, this a duplicate of #1180

**Comment by laerus** [comment]
solved on #1582
